go.mod

```diff
@@ -1,11 +1,11 @@
-go 1.18
+go 1.19
 
 module github.com/classic-terra/core
 
 require (
 	github.com/CosmWasm/wasmvm v0.16.7
-	github.com/cosmos/cosmos-sdk v0.44.8
-	github.com/cosmos/gogoproto v1.4.4
+	github.com/cosmos/cosmos-sdk v0.45.13
```

Comment:
since we are replacing the sdk anyhow, this is stylistic and just helps people know what they are working with

Reply:
Keep in mind we will be targeting the "IsOracle" based fork we implemented. You can find the pre-release version here @ https://github.com/classic-terra/cosmos-sdk/releases/tag/v0.45.13-classic

Reply:
Nvm, you sorted that out below